docs/RegisteredFilterPlugins.md

```diff
@@ -229,282 +230,279 @@ LZ4 Code:
 Although the LZ4 software is not supported by The HDF Group, it is included in The HDF Group SVN repository so that it can be tested regularly with HDF5. For convenience, users can obtain it from SVN with the following command:
    svn checkout https://svn.hdfgroup.org/hdf5_plugins/trunk/LZ4 LZ4
```

Comment:
Obsolete link and text.

Reply:
https://github.com/lz4/lz4.git

Reply:
I used this link https://github.com/HDFGroup/hdf5_plugins/tree/master/LZ4. Because it goes with the rest of the text. Should the entire text be removed, and just the link @byrnHDF suggested bu used? We do seem to have the LZ4 source code in this repo.

Reply:
From the lz4.c file in LZ4/src/lib:

   You can contact the author at :
    - LZ4 homepage : http://www.lz4.org
    - LZ4 source repository : https://github.com/lz4/lz4

Reply:
Yes, I used that one.